openbook_posts/models.py

```diff
@@ -241,7 +243,11 @@ def save(self, *args, **kwargs):
         if not self.id and not self.created:
             self.created = timezone.now()
 
-        return super(Post, self).save(*args, **kwargs)
+        post_comment = super(Post, self).save(*args, **kwargs)
```

Comment:
`post_comment` -> `post`?

Reply:
O.o it was there since time ago, probably due to copy n paste. 👍

Reply:
Oh no wait, my bad